src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7184,21 +7233,33 @@ namespace UnitarySystems {
                                     "Average",
                                     state.dataUnitarySystems->unitarySys[sysNum].Name);
 
+                SetupOutputVariable(state, "Unitary System Cooling Ancillary Electricity Energy",
+                    OutputProcessor::Unit::J,
+                    state.dataUnitarySystems->unitarySys[sysNum].m_CoolingAuxElecConsumption,
+                    "System",
+                    "Sum",
+                    state.dataUnitarySystems->unitarySys[sysNum].Name,
+                    _,
+                    "Electricity",
+                    "Cooling",
+                    _,
+                    "System");
+                SetupOutputVariable(state, "Unitary System Electricity Rate",
+                    OutputProcessor::Unit::W,
+                    state.dataUnitarySystems->unitarySys[sysNum].m_ElecPower,
+                    "System",
+                    "Average",
+                    state.dataUnitarySystems->unitarySys[sysNum].Name);
+                SetupOutputVariable(state, "Unitary System Electricity Energy",
+                    OutputProcessor::Unit::J,
+                    state.dataUnitarySystems->unitarySys[sysNum].m_ElecPowerConsumption,
+                    "System",
+                    "Sum",
+                    state.dataUnitarySystems->unitarySys[sysNum].Name);
+
                 {
                     auto const SELECT_CASE_var(state.dataUnitarySystems->unitarySys[sysNum].m_CoolingCoilType_Num);
                     if (SELECT_CASE_var == DataHVACGlobals::CoilDX_CoolingTwoSpeed) {
-                        SetupOutputVariable(state, "Unitary System Cycling Ratio",
```

Comment:
DX coil 2-speed used slightly different output variable names. Fixing that here.